packages/context/src/binding-filter.ts

```diff
@@ -59,27 +60,63 @@ export function isBindingAddress(
   return typeof bindingSelector !== 'function';
 }
 
+/**
+ * Binding filter function that uses tags
+ */
+export interface BindingTagFilter extends BindingFilter<unknown> {
```

Comment:
As far as this commit goes, it's not clear to me how is `BindingTagFilter` supposed to be used. Can you please enhance the docs and perhaps add some tests to verify the scenarios where this interface is used.

For example, you can call `filterByTag` with different arguments (a tag name, a regexp, and so on; one call per test please!) and verify that it returns `BindingTagFilter` instance with the correct value set inside `bindingTag`.